mempool/src/mempool.rs

```diff
@@ -558,17 +729,35 @@ impl Mempool {
 
     /// Gets all transaction hashes in the mempool.
     pub fn get_transaction_hashes(&self) -> Vec<Blake2bHash> {
-        self.state.read().transactions.keys().cloned().collect()
+        let state = self.state.read();
+
+        state
+            .regular_transactions
+            .transactions
+            .keys()
+            .cloned()
+            .chain(state.control_transactions.transactions.keys().cloned())
+            .collect()
```

Comment:
It would be consistent. And would also show RPC clients that those transactions are higher priority. If some where to write a fee estimator, or a confirmation estimation, that can make sense, as the control txs are included first, and the client would not need to sort them again.

But maybe that is just premature optimization :shrug: